tfs/src/main/java/hudson/plugins/tfs/rm/ReleaseWebHookHelper.java

```diff
@@ -0,0 +1,53 @@
+package hudson.plugins.tfs.rm;
+
+import hudson.plugins.tfs.TeamPluginGlobalConfig;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author Kalyan
+ */
+public final class ReleaseWebHookHelper {
+    private ReleaseWebHookHelper() {
+    }
+
+    /**
+     * Gets the release webHook configuration from global config.
+     * @return list of release webHook
+     */
+    public static List<ReleaseWebHook> getReleaseWebHookConfigurations() {
+        final TeamPluginGlobalConfig config = TeamPluginGlobalConfig.get();
+        if (config == null) {
+            throw new InternalError("Cannot load TFS global configuration");
+        }
+
+        //return config.getReleaseWebHookConfigurations();
```

Comment:
why is this commented? is this the decoupling you were referring to?

Reply:
will enable when the second PR is out.